index.html

```diff
@@ -7,24 +7,71 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="CSS/style.css">
   <title>Document</title>
+
+  <style>
+    .hide {
+      display: none;
+    }
+    .white {
+      color: aliceblue;
+    }
+  </style>
```

Comment:
- [ ] i think it would be better to move all styling to a css file, This will help keep the code base clean and easy to read